pkg/metrics/metrics.go

```diff
@@ -214,6 +217,13 @@ var (
 
 	// L7 statistics
 
+	// ProxyRedirects is the number of redirects labelled by protocol
+	ProxyRedirects = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: Namespace,
+		Name:      "proxy_redirects_total",
+		Help:      "Number of proxy redirects by application protocol",
```

Comment:
Others know more but this might be better explained with "Number of redirects installed for endpoints, labeled by protocol" since an endpoint may have more than one (per port in toPorts, I think).